extensions/phylum.ts

```diff
@@ -178,6 +222,49 @@ export class PhylumApi {
     return DenoCore.opAsync("get_job_status", jobId, ignoredPackages);
   }
 
+  /**
+   * Get job results.
+   *
+   * @param jobId - ID of the analysis job, see `PhylumApi.analyze`.
+   * @param ignoredPackages - List of packages which will be ignored in the report.
+   *
+   * @returns Raw job analysis results
+   *
+   * Job analysis results example:
+   * ```
+   * {
+   *   is_failure: false,
+   *   incomplete_packages_count: 0,
+   *   help: "The analysis contains 346 package(s) Phylum has not yet processed,\npreventing a complete risk analysis. Phylum is processing these\npackages currently and should complete soon.\nPlease wait for up to 30 minutes, then re-run the analysis.\n\nThis repository analyzes the risk of new dependencies. An\nadministrator of this repository has set requirements via Phylum policy.\n\nIf you see this comment, one or more dependencies have failed Phylum's risk analysis.",
+   *   job_link?: "https://phylum.io/projects/62a91e89-65cb-4597-8d4b-6ef119b29c5c",
```

Comment:
Why does the example have `?`s on these fields?

```suggestion
   *   job_link: "https://phylum.io/projects/62a91e89-65cb-4597-8d4b-6ef119b29c5c",
```